ui/raidboss/data/07-dt/dungeon/mistwake.txt

```diff
@@ -0,0 +1,191 @@
+### MISTWAKE
+# ZoneId: 1314
+
+hideall "--Reset--"
+hideall "--sync--"
+
+# .*is no longer sealed
+0.0 "--Reset--" SystemLogMessage { id: "7DE" } window 0,100000 jump 0
+
+#~~~~~~~~~~~~~~~~~~#
+# Treno Catoblepas #
+#~~~~~~~~~~~~~~~~~~#
+
+# IGNORED ABILITIES
+# A940 Thunder - Auto?
+# A945 Thunder II - Paired with A944
+# A946 Ray of Lightning - Paired with AF19
+
+# ALL ENCOUNTER ABILITIES
+# A93F Earthquake - Aoe/raidwide
+# A941 Thunder III - Tank buster
+# A942 Bedeviling Light - Petrify unless behind rock
+# A943 Thunder II - Lighting appears
+# A944 Thunder II - Lightning activates
+# AF19 Ray of Lightning - Line stack (breaks rocks)
+# A947 Petribreath - Front conal
+
+# TODO: get a longer run and confirm loop
+
+# The Shattered Lair will be sealed off
+1000.0 "--sync--" SystemLogMessage { id: "7DC", param1: "14EA" } window 1000,1
+1011.0 "Earthquake" Ability { id: "A93F", source: "Treno Catoblepas" }
+1020.0 "--sync--" Ability { id: "A943", source: "Treno Catoblepas" }
+1022.0 "Thunder II" Ability { id: "A944", source: "Treno Catoblepas" }
+1034.0 "Bedeviling Light" Ability { id: "A942", source: "Treno Catoblepas" }
+1040.0 "Earthquake" Ability { id: "A93F", source: "Treno Catoblepas" }
+1048.0 "Thunder III" Ability { id: "A941", source: "Treno Catoblepas" }
+1056.0 "--sync--" Ability { id: "A943", source: "Treno Catoblepas" }
+1058.0 label "treno-loop"
+1058.0 "Thunder II" Ability { id: "A944", source: "Treno Catoblepas" }
+1070.0 "Bedeviling Light" Ability { id: "A942", source: "Treno Catoblepas" }
+1078.0 "Ray of Lightning" Ability { id: "AF19", source: "Treno Catoblepas" }
+1086.0 "Petribreath" Ability { id: "A947", source: "Treno Catoblepas" }
+1096.0 "--sync--" Ability { id: "A943", source: "Treno Catoblepas" }
+1097.0 "Thunder II" Ability { id: "A944", source: "Treno Catoblepas" }
+1109.0 "Bedeviling Light" Ability { id: "A942", source: "Treno Catoblepas" }
+1115.0 "Earthquake" Ability { id: "A93F", source: "Treno Catoblepas" }
+1123.0 "Earthquake" Ability { id: "A93F", source: "Treno Catoblepas" }
+1134.0 "Thunder III" Ability { id: "A941", source: "Treno Catoblepas" }
+1144.0 "--sync--" Ability { id: "A943", source: "Treno Catoblepas" }
+1146.0 "Thunder II" Ability { id: "A944", source: "Treno Catoblepas" } forcejump "treno-loop"
+
+#~~~~~~~~~~#
+# Amdusias #
+#~~~~~~~~~~#
+
+# -ii B6F7
```

Comment:
Could you update this with the full list of ignored abilities? It's a lot easier to copy-paste a single line of ignore params than to manually type them in from the list. (Also for people who use the "click to run with params" integration in VSCode, it's nice to have that list front and center here.)

Reply:
Yep. I thought I already had but I forgot to copy it in.